frontend/src/components/grantaccess/GrantApplicationAdmin.vue

```diff
@@ -0,0 +1,182 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import { ErrorMessage, Field, Form as VeeForm } from 'vee-validate';
+import { object, string } from 'yup';
+import router from '@/router';
+import { AdminMgmtApiService } from '@/services/ApiServiceFactory';
+import { UserType, type FamApplication } from 'fam-app-acsctl-api';
+import type { FamAppAdminCreateRequest } from 'fam-admin-mgmt-api/model/fam-app-admin-create-request';
+import type { FamApplicationGetResponse } from 'fam-admin-mgmt-api/model/fam-application-get-response';
+import ApplicationSelect from '@/components/common/form/ApplicationSelect.vue';
+import Button from '@/components/common/Button.vue';
+import { IconSize } from '@/enum/IconEnum';
+import { Severity } from '@/enum/SeverityEnum';
+
+import { isLoading } from '@/store/LoadingState';
+import { setNotificationMsg } from '@/store/NotificationState';
+
+const defaultFormData = {
+    userId: '',
+    application: null as number | null,
+};
+const formData = ref(JSON.parse(JSON.stringify(defaultFormData))); // clone default input
+const formValidationSchema = object({
+    userId: string()
+        .required('User ID is required')
+        .min(2, 'User ID must be at least 2 characters')
+        .nullable(),
+    application: object().required('Application is required'),
+});
+
+const selectedApplication = ref<FamApplication>();
+const applications = ref<FamApplicationGetResponse[]>(); // clone default input
+
+onMounted(async () => {
+    applications.value = (
+        await AdminMgmtApiService.applicationsApi.getApplications()
```

Comment:
Just a note here, we should make the call to get applications in the router handler, same as the `beforeEnterGrantUserPermissionRoute`, where it grabs all roles. However since later on we'll use the new "get_my_fam_access" api to cache the result for all applications, I'm fine to leave it here, we'll change it anyway later. This is just for your reference that how we handle the on mounted call currently. 

<img width="773" alt="image" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/bf822846-e7ab-4434-82ab-e09aa81fa947">